root/series/index.tt

```diff
@@ -15,8 +15,7 @@
       IF series.type.item_entity_type == 'event';
         '<h2>' _ l('Event') _ '</h2>';
         IF entities.size;
-          INCLUDE 'components/events-list.tt'
-                  events=entities checkboxes='' no_type=1
+          React.embed(c, 'components/EventsList', { events => entities, noType => 1, seriesItemNumbers = series_item_numbers})
```

Comment:
Substituted checkboxes='' for an optional checkboxes on the component and nothing here, since the desired effect here was to have checkboxes be false anyway.